assets/scss/templates/_signup.scss

```diff
@@ -48,18 +49,36 @@
       font-family: $dp-font-family-proximanova;
     }
 
-    .signup-form {
-      .holder-small {
-        display: inline-block;
-        width: 47.4%;
+    .content-legal {
+      margin-top: 30px;
+    }
 
-        &:nth-child(1) {
-          margin-right: 4%;
+    .holder {
+      &-group {
+        display: flex;
+        flex-wrap: wrap;
+        flex-direction: column;
+      }
+
+      &-size1of1 {
+        flex: 1;
+
+        &:nth-last-child(-n+3) {
```

Comment:
Why target the last 3 rows specifically?

Reply:
Yes, I know what you mean. It's maybe replace for &:nth-child(n+2)